tokens/movistar.json

```diff
@@ -335,6 +335,11 @@
       "type": "color",
       "description": "grey3"
     },
+    "barTrackInverse": {
+      "value": "rgba({palette.black}, 0.14)",
```

Comment:
0.14, why this number? xD

Reply:
Is the number resultant of finding the nearest contrast ratio to the actual in default context using a black with opacity over background color.